clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -715,6 +722,7 @@ private PushJobSetting getPushJobSetting(VeniceProperties props) {
     pushJobSettingToReturn.repushTTLStartTimeMs = props.getLong(REPUSH_TTL_START_TIMESTAMP, System.currentTimeMillis());
     pushJobSettingToReturn.isTargetedRegionPushEnabled = props.getBoolean(TARGETED_REGION_PUSH_ENABLED, false);
     pushJobSettingToReturn.postValidationConsumption = props.getBoolean(POST_VALIDATION_CONSUMPTION_ENABLED, true);
+    pushJobSettingToReturn.isSystemSchemaReaderEnabled = props.getBoolean(SYSTEM_SCHEMA_READER_ENABLED, false);
```

Comment:
I guess we can only start using this feature after deploying the related code changes to Controller first.
Let us add such deployment order requirement in the commit messages.

Reply:
VPJ can be deployed before controller because the feature is disabled by default. But to enable the feature, this commit needs to be deployed to controller. Otherwise, the repush job will fail with `D2 service name and zk host must be provided when system schema reader is enabled` exception. I'll mention this in the commit message.